libs/damap/src/lib/components/dmp/data-storage/storage-dialog/storage-info-dialog.component.html

```diff
@@ -0,0 +1,12 @@
+<h2 mat-dialog-title>{{ data.title }}</h2>
+<mat-dialog-content>
+  <p>{{ data.description }}</p>
+  <p>
+    <a href="{{ data.link }}" target="_blank" rel="noopener">
+      {{ "Click here for more information" | translate }}
```

Comment:
Great work, everything works as intended. Good catch with the styling issue.
Just one minor thing - this should reference a translation file. Before `"dmp.steps.storage.link"` was used. Afterwards i will merge 👍